cypress/e2e/wallet-connection/switch-network.test.ts

```diff
@@ -0,0 +1,124 @@
+import { createDeferredPromise } from '../../../src/test-utils/promise'
+import { getTestSelector } from '../../utils'
+
+function getChainSelector(activeChain: string) {
+  cy.get(getTestSelector('chain-selector-logo')).invoke('attr', 'alt').should('eq', activeChain)
```

Comment:
A `should` assertion feels weird to me in a `get` function, but I can see a use for it, given cypress's "wait until the subject exists" behavior.

Reply:
maybe `waitForConnection` ?